test/examples/src.latex_simple/index.md

```diff
@@ -28,3 +28,26 @@ julia> 127 % Int8
 ```julia-repl sayhello2
 julia> function foo end;
 ```
+
+## Issue 1401
+
+1. The output is a latex tabular environment
+
+```@example
+using DataFrames
+DataFrame(i=1:3, y='A':'C')
+```
```

Comment:
As I said in https://github.com/JuliaDocs/Documenter.jl/pull/1403#discussion_r470536887, this at-example block will throw an error at the moment because we don't have DataFrames as part of the test dependencies. As we'd like to keep it that way, we'd need a custom type that would have the corresponding `show` method defined.

The `show` method could actually be defined in the same at-example block. If you're not quite sure what I mean, you can look at the image tests we have for inspiration: https://github.com/JuliaDocs/Documenter.jl/blob/master/test/examples/src/man/tutorial.md#including-images-with-mime

Reply:
Thanks. I tried to define a `show` method as you suggested, but with no success.  As DataFrames has already defined several [show](https://github.com/JuliaData/DataFrames.jl/blob/master/src/abstractdataframe/io.jl#L39-L58) methods, also see the [_show](https://github.com/JuliaData/DataFrames.jl/blob/master/src/abstractdataframe/io.jl#L250-L330) method, is it possible to import the method from there or is it valuable to define a custom type (If it valuable, I can see what I can do to help) in Documenter? Thanks. As an end user, it would be better if the tabular data can be presented in a reader friendly format.

Reply:
We wouldn't want DataFrames as a test dependency just for this. However, while I haven't tested it, something like this should work I think:

````
```@example
struct Table end
Base.show(io, ::MIME"text/latex", ::Table) = write(io, raw"""
begin{tabular}{r|ccc}
        & i & y & z\\
        \hline
        & Int64 & Char & Int64\\
        \hline
        1 & 1 & 'A' & 5 \\
        2 & 2 & 'B' & 6 \\
        3 & 3 & 'C' & 7 \\
        4 & 4 & 'D' & 8 \\
\end{tabular}
""")
Table()
```

Reply:
Example 1 does not work. 
<img width="701" alt="Example-01" src="https://user-images.githubusercontent.com/10531698/90748583-f0f23180-e304-11ea-8aa2-03f74dbe9f90.png">